lib/cocoapods-core/podfile/dsl.rb

```diff
@@ -936,6 +936,26 @@ def pre_install(&block)
         @pre_install_callback = block
       end
 
+      # This hook allows you to make changes before the project is written
+      # to disk.
+      #
+      # It receives the
+      # [`Pod::Installer`](http://rubydoc.info/gems/cocoapods/Pod/Installer/)
+      # as its only argument.
+      #
+      # @example  Customising the dependencies before integration
```

Comment:
nit "Customizing"

Reply:
followed the other definitions in this file.. I've updated them all to use proper americanism.

Reply:
@dcvz thanks, I hadn't noticed the rest of these entries. I havent written the whole code here to be aware of this.